app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -447,6 +449,23 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
         }
     }
 
+    @VisibleForTesting
+    internal fun subscribeForNewAddonsIfNeeded(
+        scope: CoroutineScope = CoroutineScope(Dispatchers.IO)
+    ) {
+        scope.launch {
+            val haveUnsupportedAddons =
+                components.addonManager.getAddons().any { it.isInstalled() && !it.isSupported() }
```

Comment:
This condition should be ` { it.isInstalled() && it.isDisabledAsUnsupported() }` because we only want to run if the user as add-ons that were disabled during migration. The supported check would also be false for Nightly overrides.

Reply:
Ok, then just checking if there are extensions that were disabled during migration will be enough

`val haveUnsupportedAddons = extensions.any { it.isUnsupported() }`